Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Controllers/NoticestoMarinersControllerTest.cs

```diff
@@ -46,6 +44,16 @@ public async Task WhenIndexIsCalled_ThenShouldReturnsExpectedView()
             Assert.AreEqual(expectedView, actualView);
         }
 
+        [Test]
+        public void WhenIndexIsCalled_ThenShouldThrowException()
```

Comment:
This name confused me.  I think a better name would be WhenIndexIsCalledAndExceptionThrownByService_ThenShouldThrowException

Reply:
Changes done.